tutorials/plot_background_statistics.py

```diff
@@ -0,0 +1,642 @@
+#      actually true:
+#
+#      .. math::
+#        \mathrm{E}(N_{\mathrm{type\ I}} / N_{\mathrm{reject}}
+#        | N_{\mathrm{reject}} > 0)
+#        \mathrm{P}(N_{\mathrm{reject}} > 0 | H_0)
+#
+# We cover some techniques that control FWER and FDR below.
+#
+# Bonferroni correction
+# ^^^^^^^^^^^^^^^^^^^^^
+# Perhaps the simplest way to deal with multiple comparisons, `Bonferroni
+# correction <https://en.wikipedia.org/wiki/Bonferroni_correction>`_
+# conservatively multiplies the p-values by the number of comparisons to
+# control the FWER.
+
+titles.append('Bonferroni')
+ts.append(ts[-1])
+ps.append(bonferroni_correction(ps[0])[1])
+mccs.append(True)
+plot_t_p(ts[-1], ps[-1], titles[-1], mccs[-1])
+
+###############################################################################
+# False discovery rate (FDR) correction
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Typically FDR is performed with the Benjamini/Hochberg procedure, which
+# is less restrictive than Bonferroni correction for large numbers of
+# comparisons (fewer type II errors) but provides less strict control of
+# errors (more type I errors).
+
+titles.append('FDR')
+ts.append(ts[-1])
+ps.append(fdr_correction(ps[0])[1])
+mccs.append(True)
+plot_t_p(ts[-1], ps[-1], titles[-1], mccs[-1])
+
+###############################################################################
+# Non-parametric resampling test with a maximum statistic
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# **Non-parametric resampling tests** can also be used to correct for multiple
+# comparisons. In its simplest form, we again do permutations using
+# exchangeability under the null hypothesis, but this time we take the
+# *maximum statistic across all tests* in each permutation to form the
+# null distribution. The p-value for each voxel from the veridical data
+# is then given by the proportion of null distribtion values
+# that were smaller.
+#
+# This method has two important features:
+#
+# 1. It controls FWER.
+# 2. It is non-parametric. Even though our initial test statistic
+#    (here a 1-sample t-test) for clustering is parametric, the null
+#    distribution for the null hypothesis rejection (cluster size
+#    distribution is indistinguishable from zero) is obtained by
+#    permutations. This means that it makes no assumptions of Gaussianity
+#    (which do hold for this example but do not in general for some types
+#    of processed neuroimaging data).
+
+titles.append('$\mathbf{Perm_{max}}$')
+out = permutation_t_test(X)[:2]
+ts.append(out[0])
+ps.append(out[1])
+mccs.append(True)
+plot_t_p(ts[-1], ps[-1], titles[-1], mccs[-1])
+
+###############################################################################
+# Clustering
+# ^^^^^^^^^^
+# Each of the aforementioned multiple comparisons corrections have the
+# disadvantage of not fully incorporating the correlation structure of the
+# data. However, by defining the connectivity/adjacency/neighbor structure
```

Comment:
Trying to think naively, "the correlation structure of the data" reads very abstract to me. The underlying concept is quite intuitive: effects cluster, in time and space. And we exploit this property of clustering in time and space (and frequency). So maybe add

 ... of the data - i.e., that effects cluster in time and space.